### PR TITLE
Added 10moon 1060 Configurations

### PR DESCRIPTION
**Pull request by Mrcubix**
# Verification:

anything before https://discord.com/channels/615607687467761684/922220574531534848/922948452139298866

# Changes

- Add TenMoonReportParser
- Add 10moon 1060 Configurations

# Note

- Doesn't meet any of the advertised specs (Only 6.52 lines per mm instead of the advertised 200 for example)
- Require multiple Feature Init
- An offset is required for pressure on aux press
- Cannot press multiple keys at the same time due to the fact bits get overwritten once a new aux key is pressed
(leading me to using bytes instead of checking for bits)

### CODE REVIEW

File: TABLETS.md [hunk 1]
```diff
@@ -27,6 +27,7 @@
 | Huion WH1409                  |     Supported     |
 | Huion WH1409 V2               |     Supported     |
 | Parblo A640 V2                |     Supported     |
+| 10moon 1060                   |     Supported     | Doesn't meet any of it's advertised specs
```
Comment (gonX) [inline TABLETS.md:30]: Grammar:
```suggestion
| 10moon 1060                   |     Supported     | Doesn't meet any of its advertised specs
```

Though, I'd prefer the comment to be reworded instead

**Review by gonX — COMMENTED** [review]
Is it our init that makes it go 4096x4096? Or does the official driver on Win Ink result in similar accuracy?

File: OpenTabletDriver.Configurations/Parsers/10moon/10moonAuxParser.cs [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+using OpenTabletDriver.Plugin;
+
+namespace OpenTabletDriver.Configurations.Parsers.TenMoon
+{
+    public struct TenMoonAuxReport : IAuxReport
+    {
+        public TenMoonAuxReport(byte[] report)
+        {
+            Log.Debug("10moon Parser", $"{BitConverter.ToString(report)}");
+
```
Comment (X9VoiD) [inline OpenTabletDriver.Configurations/Parsers/10moon/10moonAuxParser.cs]:
```suggestion
```

**Review by X9VoiD — CHANGES_REQUESTED** [review]

File: OpenTabletDriver.Configurations/Configurations/10moon/1060N.json [hunk 3]
```diff
@@ -0,0 +1,54 @@
+{
+  "Name": "10moon 1060N",
```
Comment (X9VoiD) [inline OpenTabletDriver.Configurations/Configurations/10moon/1060N.json:2]: 1060N or 1060? Update `TABLETS.md` if this is 1060N.

**Review by X9VoiD — CHANGES_REQUESTED** [review]

**Pull request merged**